README.md

```diff
@@ -41,6 +41,16 @@ If you're using Firefox locally to develop and get "import declarations may
 only appear at top level" errors, upgrade to at least Firefox 112, go to
 `about:config` and enable `dom.workers.modules.enabled`
 
+To Run via Docker in VSCode You'll need:
```

Comment:
You can "View file" in GH to see the markdown rendering:
![Screenshot from 2023-05-25 14-07-24](https://github.com/acteng/atip/assets/1664407/f7d28731-1561-4d73-a35c-4c14154d6d9a)
This could be a regular sentence, or a list using `- `